### ISSUE TITLE
Latest Android build has failed

### ISSUE THREAD

**Issue by williamd1k0** [Calinou/godot-builds-ci#13]
>I reported this on GitLab but I had not seen that the repo was just a mirror.

Hello, I'm just reporting this in case you have not noticed yet.

I don't have much experience with Android but it looks like it's just something related to licenses.

![image](https://user-images.githubusercontent.com/10428035/51446963-0570e100-1d00-11e9-85bd-ff9147406082.png)

**Comment by Calinou** [comment]
It looks like I'll have to install SDK components manually as it was done [here](https://github.com/godotengine/godot/pull/25182)…

**Comment by williamd1k0** [comment]
>Just updating the issue with the last build log

```
Cloning repository...
Cloning into '/builds/Calinou/godot-builds-ci'...
Checking out ca23f8df as master...
Skipping Git submodules setup
$ $CI_PROJECT_DIR/scripts/gitlab-ci/before_script.sh
install-info: No such file or directory for /usr/share/info/as.info.gz
install-info: No such file or directory for /usr/share/info/binutils.info.gz
install-info: No such file or directory for /usr/share/info/gprof.info.gz
install-info: No such file or directory for /usr/share/info/ld.info.gz
warning: /etc/adjtime created as /etc/adjtime.rpmnew
Setting system policy to DEFAULT
install-info: No such file or directory for /usr/share/info/history.info
install-info: No such file or directory for /usr/share/info/rluserman.info
Setting system policy to DEFAULT
Cloning into 'godot'...
-------------------------------------------------------------------------
commit 769341e180d7fb06d069ae5124e8c3caa3abfc8c
Author: Rémi Verschelde <rverschelde@gmail.com>
Date:   Mon Jan 21 11:12:51 2019 +0100

    Cleanup after @reduz :)
    
    Fixes #25172.
-------------------------------------------------------------------------
$ cd "godot/"
$ "$CI_PROJECT_DIR/scripts/gitlab-ci/build/android.sh" debug
install-info: No such file or directory for /usr/share/info/which.info.gz
ERROR: Job failed: exit code 1
```

**Comment by Calinou** [comment]
This should be fixed as of https://github.com/Calinou/godot-builds-ci/commit/373f20e5d7f5e9263b27199beb8e4db1f13a414a.
